tests/server_send.cpp

```diff
@@ -77,6 +77,28 @@ static void ssend_configure(const std::string &path)
 	ssend_servers = tests::start_nodes(cfg);
 }
 
+static void ssend_test_insert_many_keys_old_ts(session &s, int num, const std::string &id_prefix, const std::string &data_prefix)
+{
+	s.set_trace_id(rand());
+	for (int i = 0; i < num; ++i) {
+		std::string id = id_prefix + lexical_cast(i);
```

Comment:
the only available specialization of to_string is std::to_string<long long int> on gcc 4.4 (lucid)